client/src/styles/Signup-styles/SignupFormField.module.css

```diff
@@ -0,0 +1,59 @@
+/* ************* Mobile View ************* */
+.SignupFormField__input-container {
+    position: relative;
+}
+
+.SignupFormField__input {
+    height:46px;
+    border-radius: 10px;
+    width: 100%;
+    border: none;
+    outline: none;
+    box-shadow: 1.36154px 1.36154px 1.36154px rgba(0, 0, 0, 0.25);
+    font-family: var(--ff-text);
+    font-size: 17.7px;
+    color: var(--clr-dark-grey);
+    padding-left: 51px;
+}
+
+.SignupFormField__icon {
+    position: absolute;
+    top: 50%;  
+    left: 4%; 
+    transform: translateY(-50%);
+    height: 26.55px;
+    width: 26.55px;
+}
+
+.SignupFormField__error {
+    position: absolute;
+    left: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    bottom: -2px;
+    font-size: 12px;
+    color:var(--clr-light-grey);    
```

Comment:
This color is too light, it does not allow sufficient contrast to enable the user see the error message. I could not notice it easily. Please, use a color that will contrast nicely with the background color.